releasenotes/notes/wds.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: telemetry
+releaseNotes:
+- |
+  **Added** xDS workload metadata discovery to TCP metadata exchange filter as a fallback. This requires enabling `PEER_METADATA_DISCOVERY` flag on the proxy, and `PILOT_ENABLE_AMBIENT_CONTROLLERS` on the control plane.
+  **Added** a flag `PILOT_DISABLE_MX_ALPN` on the control plane to disable advertising TCP metadata exchange ALPN token `istio-peer-exchange`.
```

Comment:
May need more - or a separate doc/wiki explaining the process to move back/forth, how things interact with each other. 

Would it make sense to also add a Service annotation - for gradual opt in ? We may have cases where some services are behind east-west or other infra that requires the  peer exchange headers ( and in future as discussed JWTs or other fancy things) - while other services can use MDS. ( can be added in separate PRs, but wondering about the general adoption story in context of multi-network and other features  )

Reply:
Right now it's a fallback to existing methods, it's not taking priority. We need some burn time as a fallback before making flipping the priority - that can be done in the next version.